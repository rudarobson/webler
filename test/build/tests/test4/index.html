<!DOCTYPE html>
<!-- build:js (build/tests/test4/new/relative/path) app.js -->
	<script type="text/javascript" src="test.js"></script>
<!-- /build -->
<html>

<body>
</body>

</html>
