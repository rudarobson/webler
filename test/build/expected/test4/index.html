<!DOCTYPE html>
<script src="app.js"></script>
<html>

<body>
</body>

</html>
